packages/EasyCore/src/Doctrine/Dispatchers/DeferredEntityEventDispatcher.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Doctrine\Dispatchers;
+
+use EonX\EasyCore\Doctrine\Events\EntityCreatedEvent;
+use EonX\EasyCore\Doctrine\Events\EntityUpdatedEvent;
+use EonX\EasyCore\Interfaces\DatabaseEntityInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
```

Comment:
Please replace it with `\Psr\EventDispatcher\EventDispatcherInterface`. We will use `DeferredEntityEventDispatcher` in Lumen projects too (Lumen doesn't have the `Symfony\Contracts\EventDispatcher\EventDispatcherInterface` interface and uses `\Psr\EventDispatcher\EventDispatcherInterface` instead).

Reply:
Done